src/main/java/nextstep/ladder/domain/ladder/Height.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder.domain.ladder;
+
+public class Height {
+
+    private static final int MINIMUM_VALID_HEIGHT = 1;
+    private int height;
+
+    public Height(int heightInteger) {
+        validate(heightInteger);
+        height = heightInteger;
+    }
+
+    private void validate(int heightInteger) {
+        if(heightInteger < MINIMUM_VALID_HEIGHT) {
+            throw new IllegalArgumentException("사다리 높이는 1이상이어야 합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("사다리 높이는 " + MINIMUM_VALID_HEIGHT  + "이상이어야 합니다.");
```

와 같이 변경하면 값이 변경 됐을 때 메시지를 신경쓰지 않아도 되겠네요 ㅎㅎ